src/main/java/bio/terra/service/snapshotbuilder/query/tables/DomainOccurrence.java

```diff
@@ -0,0 +1,49 @@
+package bio.terra.service.snapshotbuilder.query.tables;
+
+import static bio.terra.service.snapshotbuilder.query.tables.Person.PERSON_ID;
+
+import bio.terra.model.SnapshotBuilderDomainOption;
+import bio.terra.service.snapshotbuilder.query.FieldVariable;
+import bio.terra.service.snapshotbuilder.query.SourceVariable;
+import bio.terra.service.snapshotbuilder.query.TablePointer;
+
+public class DomainOccurrence extends Table {
+
+  private final FieldVariable personCount;
+  private final SnapshotBuilderDomainOption domainOption;
+
+  private DomainOccurrence(SnapshotBuilderDomainOption domainOption, SourceVariable tableVariable) {
+    super(tableVariable);
+    this.domainOption = domainOption;
+    this.personCount = tableVariable.makeFieldVariable(PERSON_ID, "COUNT", "count", true);
```

Comment:
Instead of `Person.PERSON_ID`, this should define its own `PERSON_ID` field, since it's referring to the column in the domain occurrence table, not the one in the person table.

For `count`, this should use the alias name from `QueryBuilderFactory` since this is creating an alias that the result mapping code will need to use, so the two names need to agree.
```suggestion
    this.personCount = tableVariable.makeFieldVariable(PERSON_ID, "COUNT", QueryBuilderFactory.COUNT, true);
```